shuffle-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/ucx/UCX.scala

```diff
@@ -608,6 +834,12 @@ class UCX(executor: BlockManagerId, rapidsConf: RapidsConf) extends AutoCloseabl
 
   override def close(): Unit = {
     onWorkerThreadAsync(() => {
+      amRegistrations.forEach { (activeMessageId, _) =>
+        logInfo(s"Removing Active Message registration for " +
```

Comment:
Do we want this at info or debug level?

Reply:
There should be 4 of these. But yeah I'll move to debug.